arch/x86/include/asm/kvm_pkvm.h

```diff
@@ -107,6 +107,30 @@ enum pkvm_hc {
 	MAX_PKVM_HYPERCALLS,
 };
 
+#define PKVM_FN_ARG_NUM		4
+
+union pkvm_hc_data {
```

Comment:
nit: maybe a better name would be something like `pkvm_hc_params` (to reflect its purpose)?

Reply:
Those are "output parameters". :) And actually, in handle_vmcall() and pkvm_handle_host_hypercall() we are using `pkvm_hc_data` for any hypercall parameters?

But that's a nit, I'm fine with keeping the name `pkvm_hc_data` too.

Reply:
OK. If so, I prefer to keep use `pkvm_hc_data`